code/src/Components/Summary.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+function refreshPage() {
+  window.location.reload(false);
+}
+
+const Summary = (props) => {
```

Comment:
The same here (and on all props): You don't need to write props and deconstruct them into variables, just add them in the ({radio, genre, number, recommendation})